### PR TITLE
added suggested replies doc

### LINKED ISSUE #1363: Suggested Replies: Docs
Now that we have suggested replies we should include it in the docs:

- Create a new page in UI Section
- Add a TLDR
- Add a paragraph describing it
- Add a gif showing it in the inbox
- Add a link to the Rasa Integration/how can it be filled?

### PR DESCRIPTION
**Pull request by AudreyKj**
closes #1363

### CODE REVIEW

File: docs/docs/ui/suggestedReplies.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+The screenshot below shows a sample conversation in the UI of an Airy Core instance: two suggested replies have been added to the contact's last message ('Great that it worked. Is there anything else you need?' and 'Have a nice day!'). The user clicks on the Suggestions button at the top left of the conversation's input and selects the suggested reply 'Have a nice day!'.
+
+<img alt="suggestedReplies" src={useBaseUrl('img/ui/suggested-replies.gif')}/>
+
+## API request to add suggested replies
+
+<ButtonBox
+icon={<BoltSVG />}
+title='Channels endpoint'
```
Comment (lucapette) [inline docs/docs/ui/suggestedReplies.md:26]:
```suggestion
title='Suggest replies endpoint'
```

File: docs/docs/ui/suggestedReplies.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import TLDR from "@site/src/components/TLDR";
+import ButtonBox from "@site/src/components/ButtonBox";
+import BoltSVG from "@site/static/icons/bolt.svg";
+import RasaSVG from "@site/static/icons/rasa.svg";
+
+<TLDR> Suggested Replies allow you to add predefined responses to conversation messages, thus improving the response time and the overall communication experience.</TLDR>
+
+Suggested Replies are text replies that are added to one of the last 5 messages in a conversation. They are added via API request and only support text content.
```
Comment (AitorAlgorta) [inline docs/docs/ui/suggestedReplies.md:14]: I think it supports any type of message

File: docs/docs/api/endpoints/suggest-replies.mdx [hunk 3]
```diff
@@ -0,0 +1,47 @@
+`POST /messages.suggestReplies`
+
+Suggest a set of replies for a given message id. UI clients can show these to agents to make responding to user inquiries
```
Comment (lucapette) [inline docs/docs/api/endpoints/suggest-replies.mdx:3]:
```suggestion
Suggest a set of replies for a given message id. UI clients can then show these replies to agents  to help them responding user inquiries
```

File: docs/docs/ui/suggestedReplies.md (hunk 2 above)
Comment (lucapette) [inline docs/docs/ui/suggestedReplies.md:14]:
```suggestion
Suggested Replies are text replies that you can add to any message via the HTTP API. The suggested replies of the last 5 messages of a given conversation are considered for suggestion.
```

File: docs/docs/ui/suggestedReplies.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+As soon as suggested replies have been successfully added to a message (i.e. the API request returned a 200 response), a Suggestions button appears in the conversation's UI, at the top left of the input, enabling the user to choose a suggested reply as answer.
```
Comment (lucapette) [inline docs/docs/ui/suggestedReplies.md:16]:
```suggestion
As soon as suggested replies have been successfully added to a message (i.e. the API request returned a 200 response), a suggestions button appears at the top left of the input in the messages list of conversation's UI, enabling the user to choose a suggested reply as answer.
```

**Review by lucapette — CHANGES_REQUESTED** [review]

**Review by lucapette — APPROVED** [review]

**Pull request merged**